modules/dxp/apps/portal-rules-engine-drools/portal-rules-engine-drools/build.gradle

```diff
@@ -9,7 +9,6 @@ dependencies {
 	compileInclude group: "org.mvel", name: "mvel2", version: "2.1.0.drools16"
 
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
-	compileOnly group: "com.thoughtworks.xstream", name: "xstream", version: "1.4.17"
```

Comment:
@rafaprax Why you remove this? If there is a reference to the classes of "com.thoughtworks.xstream" inside this module, we should update this, if not, this is not needed even if you did not remove those jars in portal/lib, then you should do this related clean up first.